src/doc/main.lisp

```diff
@@ -19,18 +20,32 @@
 
 (in-package #:coalton/doc)
 
+(defvar *coalton-github*
+  "https://github.com/coalton-lang/coalton")
+
 (defun head-revision ()
-  "Look up the most recent commit, either in the env var GITHUB_SHA which is provided by github CI, or by calling out to git rev-parse."
+  "Get the current commit by checking the environment variable GITHUB_SHA set by GitHub CI. If it is unset, use the output of git rev-parse instead."
   (or (uiop:getenv "GITHUB_SHA")
       (string-trim '(#\Newline)
                    (uiop:run-program "git rev-parse HEAD" :output ':string))))
 
+;;;  In GitHub CI, the environment variable GITHUB_WORKSPACE names the
+;;;  directory containing the directory into which Coalton is cloned
+;;;  during build, so '/coalton' is appended to that, resulting in the
+;;;  path /__w/coalton/coalton/coalton, like summoning Beetlejuice.
```

Comment:
Betelgeuse, Betelgeuse, Betelgeuse, but I'm not sure if humor is permissible in comments.

Reply:
Then why is it allowed in docstrings!

https://github.com/coalton-lang/coalton/blob/6f0753b0a548ce40ef6f05d5c5ab0287226408f3/library/hashtable.lisp#L71

Reply:
levity eradicated